jpsreport/Analysis.h

```diff
@@ -32,21 +32,20 @@
 #ifndef ANALYSIS_H_
 #define ANALYSIS_H_
 
-#include "IO/OutputHandler.h"
 #include "general/ArgumentParser.h"
 #include "general/Macros.h"
 #include "geometry/Building.h"
 #include "methods/ConfigData_DIJ.h"
 #include "methods/PedData.h"
 
+#include <Logger.h>
```

Comment:
Why include Logger.h here and not in Analysis.cpp?

Reply:
Done